pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/AbstractLombokAwareRule.java

```diff
@@ -82,11 +91,12 @@ protected boolean hasClassLombokAnnotation() {
     /**
      * Checks whether the given node is annotated with any lombok annotation.
      * The node can be any node, e.g. class declaration or field declaration.
-     * 
+     *
      * @param node
      *            the node to check
      * @return <code>true</code> if a lombok annotation has been found
      */
+    @Deprecated
```

Comment:
you should add a `@deprecated` on the javadoc too, indicating what to use instead